Mathlib/LinearAlgebra/QuadraticForm/TensorProduct.lean

```diff
@@ -27,7 +27,7 @@ section CommRing
 variable [CommRing R] [CommRing A]
 variable [AddCommGroup M₁] [AddCommGroup M₂] [AddCommGroup N₁] [AddCommGroup N₂]
 variable [Algebra R A] [Module R M₁] [Module A M₁] [Module R N₁] [Module A N₁]
-variable [SMulCommClass R A M₁] [IsScalarTower R A M₁] [SMulCommClass R A N₁] [IsScalarTower R A N₁]
```

Comment:
It surprises me that nothing else complains that this variable isn't needed. The symmetry breaking is a bit jarring, but I guess if Lean say it's not needed then we have no excuse to keep it!